hat/app/org/hatdex/hat/api/controllers/Authentication.scala

```diff
@@ -223,4 +225,100 @@ class Authentication @Inject() (
         Future.successful(Unauthorized(Json.toJson(ErrorMessage("Invalid Token", "Token does not exist"))))
     }
   }
+
+  /*
+    BEGIN: hat-claim
+    Entire Section below is on HAT Claim.
+    TODO: Refactor to New File?
+   */
+  /**
+   * Sends an email to the owner with a link to claim the hat
+   */
+  def claim(applicationId: String): Action[AnyContent] = SecuredAction(WithRole(Owner())).async { implicit request =>
```

Comment:
It's unnecessary to use owner token authentication. Instead, it should take applicationId and ownerEmail as input and as long as owner email matches, proceed.

Reply:
1. Regarding token authentication, what shall we used instead?
2. Re applicationId and userEmail, implemented as suggested.

Reply:
Resolved with email validation similar to forgot password endpoint